lib/rollup/index.js

```diff
@@ -13,13 +13,11 @@ const plugin = (config) => async (files, metalsmith, done) => {
   const file = normalize(config.input)
   const map = `${file}.map`
 
-  // Resolve paths from Metalsmith source
-  config.input = resolve(metalsmith.source(), file)
-  config.output.file = resolve(metalsmith.destination(), file)
```

Comment:
Turns out `config.output.file` is only needed by **bundle.write()** not **bundle.generate()**